subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileIntegrationTest.groovy

```diff
@@ -41,7 +41,7 @@ class JavaCompileIntegrationTest extends AbstractPluginIntegrationTest {
         file("src/main/java/Foo.java") << "public class Foo {}"
 
         when:
-        run("compileJava")
+        run("compileJava", "-Dorg.gradle.vfs.debug=true")
```

Comment:
I suppose this is a leftover change from trying out the new flag. Can you please remove it?

Reply:
Removed.